templates/around/map.html

```diff
@@ -12,17 +12,68 @@
+            <path d="M4445.2,2265.6v-554.3h-565.4h-565.4v-554.3V602.6h565.4h565.4V37.2v-565.4h554.3h554.4V37.2v565.4h565.4h565.4l-4.5,549.9l-6.6,547.7l-558.8,6.7l-561,4.4v554.3v554.3h-554.4h-554.3V2265.6z"></path>
+          </g>
+        </svg>
+      </button>
+    </div>
+    </div>
     <div id=poi-cards></div>
-  </div>
+    <div id=poi-form>
+      <form action= method=post>
+        <button id=close-poiForm class=display-form type=button>
+          <svg viewBox="0 0 128 128">
+            <circle cx="64" cy="64" r="64"/>
+            <path d="M100.3,90.4L73.9,64l26.3-26.4c0.4-0.4,0.4-1,0-1.4l-8.5-8.5c-0.4-0.4-1-0.4-1.4,0L64,54.1L37.7,27.8  c-0.4-0.4-1-0.4-1.4,0l-8.5,8.5c-0.4,0.4-0.4,1,0,1.4L54,64L27.7,90.3c-0.4,0.4-0.4,1,0,1.4l8.5,8.5c0.4,0.4,1.1,0.4,1.4,0L64,73.9  l26.3,26.3c0.4,0.4,1.1,0.4,1.5,0.1l8.5-8.5C100.7,91.4,100.7,90.8,100.3,90.4z"/>
+          </svg>    
+        </button>
+        <h2>Share your place</h2>
+        <ul>
+          <li>
+            <label for=name>Name place</label>
+            <input name=name type=text>
+          </li>
+          <li>
+            <label for=wifi>Wifi quality</label>
+            <input name=wifi min=0 max=5 value=3 type=range>
+          </li>
+          <li>
+            <label for=power-available>Power is available</label>
+            <input name=power type=checkbox>
+          </li>
+          <li>
+            <label for=type>Type of place</label>
+            <select name=type>
+                <option value=coworking>Coworking space</option>
+                <option value=coffeeshop>Coffeeshop / Restaurant</option>
+                <option value=other>Library / Museum / other</option>
+            </select>
+          </li>
+          <li>
+            <label for=comment>Anything to say ?</label>
+            <textarea name=comment type=text></textarea>
+          </li>
+          <li>
+            <label for=author>Your nickname</label>
+            <input name=author type=text>
+          </li>
+        </ul>
+        <button id=save-poiForm role=button>Save</button>
+      </form>
+    </div>
 
   <script>
     var currentLatLng = {{ current_user.current_location or [] }}
     var locations = [
     {% for user in users %}
       {
         location: {{ user.current_location }},
-        user: "{{ user }}"
+        user: {{ user }}
```

Comment:
I don't think you tested this because JS is broken here.
`{{ user }}` is python, writing in a `<script>` section, therefore it's writing some JS.
this would generate `user: John` where _John_ would be undefined.
You want to keep the quotes `user: "John"`.
Avoid single quotes in this specific case as we may have single quotes in user names (much less chance to have double quotes though it could happen).